quarkus/config-api/src/main/java/org/keycloak/config/CachingOptions.java

```diff
@@ -50,9 +50,8 @@ public enum Stack {
 
     public static final Option<Stack> CACHE_STACK = new OptionBuilder<>("cache-stack", Stack.class)
             .category(OptionCategory.CACHE)
-            .expectedValues(List.of())
-            .description("Define the default stack to use for cluster communication and node discovery. This option only takes effect "
-                    + "if 'cache' is set to 'ispn'. Default: udp. Built-in values include: " + Stream.of(Stack.values()).map(Stack::name).collect(Collectors.joining(", ")))
+            .expectedValues(false)
+            .description("Define the default stack to use for cluster communication and node discovery. This option only takes effect if 'cache' is set to 'ispn'. Default: udp.")
```

Comment:
Should we mark `udp` as the formal default value in the Options instead of just mentioning it in the desc? It'd help with formatting especially in the guide.

Reply:
Yep, it'd be better to do it, but there's a bigger issue around defining the default value. When the default value is specified by this, it is not possible to leverage the ability to use the stack specified in the ISPN/JGroups cache file[1]. 

In the doc[1], there are these sentences: 
_"By default, the value set to the cache-stack option has precedence over the transport stack you define in the cache configuration file. If you are defining a custom stack, make sure the cache-stack option is not used for the custom changes to take effect."_

That means, when we specify the default value, the default value will always be present (`udp` in this case), and the custom stack defined in the cache file will not be used.

I think it's not worth providing any comprehensive handling of this case. @vmuzikar WDYT?

Relevant issue: https://github.com/keycloak/keycloak/issues/10052

[1] https://www.keycloak.org/server/caching#_custom_transport_stacks

Reply:
Thanks for pointing that out. I agree it's not worth it to over-engineer this. Let's keep as is.